GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/StorageClient.UploadObject.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Upload;
+using Google.Common;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Object = Google.Apis.Storage.v1.Data.Object;
+
+namespace Google.Apis.Storage.v1.ClientWrapper
+{
+    public partial class StorageClient
+    {
+        /// <summary>
+        /// Uploads the data for an object in storage synchronously, from a specified stream.
+        /// </summary>
+        /// <param name="bucket">The name of the bucket in which to store the object. Must not be null.</param>
+        /// <param name="objectName">The name of the object within the bucket. Must not be null.</param>
+        /// <param name="contentType">The content type of the object. Must not be null.</param>
```

Comment:
Another thing is, wouldn't be useful to have an overload that specifies a default value, `application/octet-stream` or something like that?